<?xml version="1.0" encoding="UTF-8"?>
<testng-results total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2022-11-18T15:17:00Z" name="Suite" finished-at="2022-11-18T15:17:00Z" duration-ms="42">
    <groups>
    </groups>
    <test started-at="2022-11-18T15:17:00Z" name="Test" finished-at="2022-11-18T15:17:00Z" duration-ms="42">
      <class name="test.PhpTravels.TestNGAnnotations">
        <test-method is-config="true" signature="beforeSuite()[pri:0, instance:test.PhpTravels.TestNGAnnotations@4f67eb2a]" started-at="2022-11-18T20:47:00Z" name="beforeSuite" finished-at="2022-11-18T20:47:00Z" duration-ms="10" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeSuite -->
        <test-method is-config="true" signature="beforeTest()[pri:0, instance:test.PhpTravels.TestNGAnnotations@4f67eb2a]" started-at="2022-11-18T20:47:00Z" name="beforeTest" finished-at="2022-11-18T20:47:00Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:test.PhpTravels.TestNGAnnotations@4f67eb2a]" started-at="2022-11-18T20:47:00Z" name="beforeMethod" finished-at="2022-11-18T20:47:00Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="test()[pri:0, instance:test.PhpTravels.TestNGAnnotations@4f67eb2a]" started-at="2022-11-18T20:47:00Z" name="test" finished-at="2022-11-18T20:47:00Z" duration-ms="13" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[value of i is not equal to 3 expected [2] but found [3]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: value of i is not equal to 3 expected [2] but found [3]
	at org.testng.Assert.fail(Assert.java:89)
	at org.testng.Assert.failNotEquals(Assert.java:489)
	at org.testng.Assert.assertEquals(Assert.java:118)
	at org.testng.Assert.assertEquals(Assert.java:365)
	at test.PhpTravels.TestNGAnnotations.test(TestNGAnnotations.java:40)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:568)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:715)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:907)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1237)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:767)
	at org.testng.TestRunner.run(TestRunner.java:617)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
	at org.testng.SuiteRunner.run(SuiteRunner.java:240)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:51)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:85)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1197)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1122)
	at org.testng.TestNG.run(TestNG.java:1030)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test -->
        <test-method is-config="true" signature="afterTest()[pri:0, instance:test.PhpTravels.TestNGAnnotations@4f67eb2a]" started-at="2022-11-18T20:47:00Z" name="afterTest" finished-at="2022-11-18T20:47:00Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
        <test-method is-config="true" signature="afterSuite()[pri:0, instance:test.PhpTravels.TestNGAnnotations@4f67eb2a]" started-at="2022-11-18T20:47:00Z" name="afterSuite" finished-at="2022-11-18T20:47:00Z" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterSuite -->
      </class> <!-- test.PhpTravels.TestNGAnnotations -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
